SwiftDraw/LayerTree.Color.swift

```diff
@@ -100,14 +110,14 @@ extension LayerTree.Color {
     switch self {
     case .none:
       return .none
-    case let .rgba(r: r, g: g, b: b, a: _, space):
+    case let .rgba(r: r, g: g, b: b, a: a, space):
       return .rgba(r: r,
                    g: g,
                    b: b,
-                   a: alpha,
+                   a: alpha * a,
```

Comment:
I made the two values multiply instead of overwriting with `alpha`. `a` would be 1 for all other color types, so the behaviour remains the same as before.